src/overseer/heartbeat.clj

```diff
@@ -87,6 +87,14 @@
           (timbre/warn (str "Resetting: " (string/join ", " (map :job/id jobs))))
           @(d/transact conn dead-job-txns))
         (Thread/sleep (+ (config/heartbeat-sleep-time config) (sleep-stagger))))
+      (catch java.util.concurrent.ExecutionException ex
+        (if (= :db.error/cas-failed
+               (->> ex
+                    Throwable->map
+                    :data
+                    :db/error))
+          (timbre/info "Heartbeat monitor CAS failed; ignoring.")
```

Comment:
I don't have a super strong preference, nonblocker, but to me info is like "starting this" or "progress X", whereas warn is like "an exception *did* occur but its ok...." etc